### ISSUE TITLE
Unable to create multiple instances of PDF viewer in a single page

### ISSUE THREAD

**Issue by xtephan** [VadimDez/ng2-pdf-viewer#513]
It is not possible to create 2 different pdf viewer in 1 page. One of them will be rendered file while the other will just have a loading spinner. 
An example can be found here: https://stackblitz.com/edit/angular-eg93wm

It seems that `[show-all]="true"` is somehow at fault here, as changing it to `[show-all]="false"` will make it work

##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [X] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

**Comment by cyberdante** [comment]
Has there been any progress on this? This is an unfortunately common scenario that needs to be solved.

**Comment by xtephan** [comment]
@cyberdante  the workaround I found was to wait until the first PDF finished loading before creating/setting a source for the second one. It seems to work pretty reliably for me.

**Comment by cyberdante** [comment]
Thanks for the workaround, @xtephan. I have that working as well on my current use case but was hoping for an eventual solution. Thank you!

**Comment by VadimDez** [comment]
This seems to be fixed with version `6.0.0`
